torch/csrc/autograd/functions/convolution.h

```diff
@@ -29,6 +29,7 @@ struct ConvParams {
   std::vector<int> output_padding;
   int groups;
   bool benchmark;
+  bool deterministic;
```

Comment:
This field is never set anywhere, and since it's a primitive type, it is not initialized (which means it will randomly be true and false at different times).

Reply:
resolved!